data/models/noteModel.js

```diff
@@ -0,0 +1,138 @@
+const db = require('../dbConfig.js');
+
+const get = async (id) => {
+    if (id) {
+        const note = await db('notes').where('id', id);
+        if (note.length) {
+            return note[0];
+        } else {
+            const e = new Error("id does not exist");
+            e.name = "InvalidID";
+            throw e;
+        }
+    }
+
+    const notes = await db('notes');
+    return notes;
+};
+
+const insert = async (note) => {
+
+    // check for missing note object
+    if (typeof note === 'undefined') {
```

Comment:
I don't think you need this check 🤔 is there ever a case where this would happen?